Sources/Leaf/Application+Leaf.swift

```diff
@@ -51,9 +51,12 @@ extension Application {
             }
         }
 
-        public var files: LeafFiles {
+        public var files: LeafSource {
```

Comment:
Isn't this a breaking change?

Reply:
Sorrrrrrt of? 🙃 I specifically avoided renaming `files` to avoid breaking in the refactoring sense to Vapor+Leaf, but the commensurate LeafKit PR typealiases the name change. Given that the protocol has no public methods in any case - I'm not sure this is a huge issue.

If someone has built a custom adherent to LeafFiles, the combined PRs *are* breaking regardless in LeafKit as the protocol is updated to support the escaping control flag, regardless of the name change, and default implementations will always error to force upgrading (though that's a very simple change to the main function call signature).

It's a minor matter but as I intend to add support for Fluent-backed template stores, I'd like to abstract the name, and the concept of "template names", away from being tied to filesystems as much as possible for clarity. Ideally `files` would additionally become an array or dictionary of Sources in a future release so that multiple stores can be used (eg, first source being developer-tested templates on filesystem, second source if template isn't found there being database-backed end-user dynamic templates)

More details on the wholesale changes at the [LeafKit PR](https://github.com/vapor/leaf-kit/pull/55)